vm_method.c

```diff
@@ -295,6 +297,8 @@ void
 rb_clear_method_cache_all(void)
 {
     rb_objspace_each_objects(invalidate_all_cc, NULL);
+
+    rb_yjit_invalidate_all_method_lookup_assumptions();
```

Comment:
Oof, when and how often is this typically called during the execution of a Ruby program?